config/routes.rb

```diff
@@ -6,75 +6,83 @@
   match "/500", :to => "errors#internal_server_error", :via => :all
   match "/503", :to => "errors#service_unavailable", :via => :all
 
-  scope :apply do
-    resource :guide,              only: :show
-    resource :terms,              only: :show
-    resource :cookies,            only: :show
-    resource :claim_review,       only: %i<show update>, path: :review
-    resource :pdf,                only: :show
-    resource :claim_confirmation, only: :show, path: :confirmation
-
-    resource :claim, only: :create, path: "/" do
-      resource :payment, only: %i<show update>, path: :pay do
-        member do
-          %i<success decline>.each do |result|
-            get result, to: "payments##{result}", as: result
+  scope "(:locale)", locale: /en|cy/ do
```

Comment:
Can this default to en if its not present ?  I did this in the calculator but cannot remember how !
If we dont do this, if someone is in the middle of a session when we deploy it, they will get a 500 error on the following page.
Also, if anyone has things bookmarked, they will break
Any google results that are cached would also break